.travis.yml

```diff
@@ -1,24 +1,13 @@
 language: node_js
-sudo: required
+sudo: false
 node_js:
-  - "4.1"
-before_install:
-  - "sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test"
-  - "sudo apt-get update -qq"
-  - "export DISPLAY=:99.0"
-  - "sh -e /etc/init.d/xvfb start"
-
-install:
-  - "sudo apt-get install -qq gcc-4.8 g++-4.8"
-  - "sudo update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-4.8 90"
-  - "npm install"
+  - "4.2.2"
 
-env:
-  - CHROME_BIN=/usr/bin/chromium-browser COVERALLS_SERVICE_NAME=travis-ci COVERALLS_REPO_TOKEN=sj8SOxUBGaVeFGSY9ryrC2eTFpVjCPHJl
```

Comment:
I think it's still good to send coveralls, but we can do that after the merge :+1: